tests/csm/rest/test_capacity_usage.py

```diff
@@ -1968,3 +2018,914 @@ def test_34723(self):
+                assert self.csm_obj.verify_degraded_capacity(
+                  resp, healthy=write_bytes_mb, degraded=healthy_count, critical=0, damaged=0, err_margin=10,
+                  total=total_written)
+                self.log.info(
+                  "[End] Fetch degraded capacity on Consul with 1 Node failure")
+                self.log.info(
+                 "[Start] Fetch degraded capacity on HCTL with 1 Node failure")
+
+                resp = self.hlth_master.hctl_status_json()["bytecount"]
+                cap_df.loc[index]["hctl_healthy"] = resp["healthy"]
+                cap_df.loc[index]["hctl_degraded"] = resp["degraded"]
+                cap_df.loc[index]["hctl_critical"] = resp["critical"]
+                cap_df.loc[index]["hctl_damaged"] = resp["damaged"]
+                assert self.csm_obj.verify_degraded_capacity(
+                   resp, healthy=write_bytes_mb, degraded=healthy_count, critical=0, damaged=0, err_margin=10,
+                     total=total_written)
+                self.log.info(
+                  "[End] Fetch degraded capacity on HCTL with 1 Node failure")
+
+                self.log.info("[Start] Fetch degraded capacity on CSM with 1 Node failure")
+                resp = self.csm_obj.get_degraded_capacity()
+                assert resp.status_code == HTTPStatus.OK, "Status code check failed."
+                resp = resp.json()["bytecount"]
+                cap_df.loc[index]["csm_healthy"] = resp["healthy"]
+                cap_df.loc[index]["csm_degraded"] = resp["degraded"]
+                cap_df.loc[index]["csm_critical"] = resp["critical"]
+                cap_df.loc[index]["csm_damaged"] = resp["damaged"]
+                assert self.csm_obj.verify_degraded_capacity(
+                   resp, healthy=write_bytes_mb, degraded=healthy_count, critical=0, damaged=0, err_margin=10,
+                        total=total_written)
+                self.log.info(
+                  "[End] Fetch degraded capacity on CSM with 1 Node failure")
+                self.log.info("-----------------------Step 5 Ends--------------------------")
+                self.log.info("Check ended for k=1, i.e N1 pod failure")
+            elif node==0:
+                self.log.info("Check started for k=0, i.e N2 pod failure")
+                self.log.info("[Start] Shutdown the data pod safely by making replicas=0")
+                resp = self.master.create_pod_replicas(num_replica=0, deploy=deploy_name[1])
+                assert_utils.assert_false(resp[0],
+                          f"Failed to delete pod {deploy_name} by making replicas=0")
+                self.log.info("[End] Successfully shutdown/deleted pod %s by making replicas=0",
+                     deploy_name)
+                self.log.info("-----------------------Step 6 Starts--------------------------")
+                self.log.info(
+                 "[Start] Fetch degraded capacity on Consul with 2 Node failure")
```

Comment:
it is self.kvalue - node failures and not 2 node failure

Reply:
done